Xpra is a good piece of software except that some features are not
useful only to introduce unnecessary dependencies.

Forwarded: no need.

Index: xpra-4.4.6/xpra/scripts/main.py
===================================================================
--- xpra-4.4.6.orig/xpra/scripts/main.py
+++ xpra-4.4.6/xpra/scripts/main.py
@@ -64,7 +64,7 @@ CLIPBOARD_CLASS = os.environ.get("XPRA_C
 WAIT_SERVER_TIMEOUT = envint("WAIT_SERVER_TIMEOUT", 90)
 CONNECT_TIMEOUT = envint("XPRA_CONNECT_TIMEOUT", 20)
 OPENGL_PROBE_TIMEOUT = envint("XPRA_OPENGL_PROBE_TIMEOUT", 5)
-SYSTEMD_RUN = envbool("XPRA_SYSTEMD_RUN", True)
+SYSTEMD_RUN = False
 VERIFY_X11_SOCKET_TIMEOUT = envint("XPRA_VERIFY_X11_SOCKET_TIMEOUT", 1)
 LIST_REPROBE_TIMEOUT = envint("XPRA_LIST_REPROBE_TIMEOUT", 10)
 
